types/js-captcha/index.d.ts

```diff
@@ -1,7 +1,42 @@
-// Type definitions for js-captcha 1.1
+// Type definitions for js-captcha 1.3
 // Project: https://github.com/robiveli/js-captcha
 // Definitions by: PikachuEXE <https://github.com/pikachuexe>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import * as jCaptcha from "./dist/js/jCaptcha";
+declare namespace jCaptcha {
+    /**
+     * Options for the base jCaptcha class. Derived from https://github.com/robiveli/js-captcha#options
+     */
+    interface Options {
+        el?: string;
+
+        requiredValue?: string;
+
+        resetOnError?: boolean;
+        focusOnError?: boolean;
+        clearOnSubmit?: boolean;
+
+        canvasClass?: string;
+        canvasStyle: {
+            width?: number;
+            height?: number;
+
+            font?: string;
+            fillStyle?: string;
+            textAlign?: string;
+            textBaseline?: string;
+        };
+
+        callback?: (response: "success" | "error", captcha: Element, numberOfTries: number) => void;
+    }
+}
+
+declare class jCaptcha {
+    constructor(options?: jCaptcha.Options);
+
+    validate(): void;
+    reset(): void;
+}
+
+export = jCaptcha;
```

Comment:
```suggestion
export as namespace jCaptcha;
export = jCaptcha;
```
I think this is an UMD module, used in the browser:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html
https://github.com/robiveli/js-captcha/blob/master/dist/js/index.js
https://unpkg.com/js-captcha@1.3.0/dist/js/index.js
(if accepted, please verify it works locally)